src/platform.ts

```diff
@@ -34,33 +56,12 @@ export function getCurrentPlatform() {
         const text = child_process.execSync('cat /etc/os-release').toString();
         const lines = text.split('\n');
 
-        function getValue(name: string) {
```

Comment:
Is this change because TypeScript wasn't working for you? This should probably go in as a separate PR.